docs/DeveloperGuide.md

```diff
@@ -296,6 +298,30 @@ Design considerations:
 is an existing method that already gets the updated Person list, hence, we have decided to get the portfolio
 of each client through this Person list as each client will have their own portfolio.
 
+### 4.3. Sort Command
+
+#### Current Implementation
+
+The sort command mainly relies on the following classes:
+* `AddressBookParser`
+* `SortCommand`
+* `SortCommandParser`
+* `ParserUtil`
+* `Model`
+* `UniquePersonList`
+
+1. The user executes the `sort` command while providing a sorting parameter `sortParam` as an argument.
+2. `AddressBookParser#parseCommand` is called, which creates and returns a new `SortCommandParser` with the
+   provided `sortParam`.
+3. `SortCommandParser#parse()` is called, which creates and returns a new `SortCommand` with the given `sortParam`.
+4. `SortCommand#execute()` is called.
+    1. `SortCommand#execute()` will pass `sortParam` to `model#sort`.
+    2. `model#sort` will call `UniquePersonList#sort` to sort client list.
```

Comment:
Should we mention the intermediate step of `Model#sort` calling `AddressBook#sort` which then calls `UniquePersonList#sort` here?

Reply:
Other than this, looks good to merge! 👍

Reply:
Will fix and merge!